### DISCUSSION TITLE
這些暖身運動該被淘汰了

### DISCUSSION THREAD

**Discussion by Elaine2899** [1101-GS4719-C/Python-Community-GS4719-C#1504] [General]
隨著科技進步，人類的知識也在不斷更新，對於"暖身"的相關知識也是。
從小體育老師在前面帶操我們就跟著做，但那些暖身姿勢真的有"暖"到嗎？
哪些是該被淘汰的姿勢呢？
1. 轉脖子：我們的脖子可以正常做到6個動作，也就是前彎後仰、左右側倒以及左右旋轉。而在轉脖子時，我們做的是複合式的動作，導致每個角度都沒辦法拉到底。另外在側後島的動作時，會摩擦到我們傳遞訊息的神經根，導致脖子往下到手的一條線傳導痛的感覺，甚至會有痠痛麻的感，也可能壓迫到上頸椎部分的椎動脈，而壓迫到血管，進而引起頭暈頭痛。
2. 腰部環繞：這個暖身運動可以說是相當沒有意義，因為腰部肌群多由背部或是從側邊連接而來的肌群，因此這樣的環繞運動並很難拉伸到我們腰部的肌群。
3. 膝蓋環繞：這個動作同樣沒有意義，我們的膝蓋只能做兩個動作，彎取和伸直，因此這個動作雖然不會造成什麼傷害，但也是好像有做什麼，其實什麼都沒做
4. 肩膀旋轉：因為我們的肩膀並不能旋轉360度，大約只能往上180度、往後50度，因此往上超過180度時，會讓棘上肌被摩擦到，造成肌腱容易受傷甚至撕裂等問題。尤其對於有點駝背的人來說，他們的夾擠程度會變得更加嚴重

以上是幾個常見但應該可以被淘汰或取代的動作了！

我自己在老師做操的時候通常蠻認真的，因為我喜歡身體被伸展開來的感覺，但做到上述動作時是真的不知道我拉到哪裡了，尤其是肩膀旋轉的部分會覺得不太舒服，看了這個影片才知道原來這些已經是舊的、該被淘汰的身動作了！
希望大家運動前也會好好暖身避免運動傷害哦！

參考資料：https://youtu.be/yMREgzy4Kwc

**Comment by brian431** [comment]
原來是這樣，大多旋轉累的動作都對拉伸沒什麼幫助，畢竟人的肌肉不是設計來旋轉的。跟著老師做了好幾年的暖身竟然沒用，太難過了。

**Comment by pudding0803** [comment]
我從國中就聽說過第一個會傷害到脖子，但大部分的老師都還是會叫大家做（然後我都偷偷不做？）
另外之前在做二、三的時候，我就一直覺得沒什麼感覺，原來是真的沒有用🤣
第四個我現在才知道！！以後會注意的🥲

不過就我看來，也有越來越多老師知道哪些做了反而是不好的，但無論是老師還是學生，都應該要了解一下，以保護自己的身體

**Comment by bestbanana295** [comment]
原來是這樣，看來我這些年來做的暖身原來都有一點瑕疵，下此如果我要運動的話會避開這些運動來去做，蝦蝦你的小知識補帖，受益良多。我覺得暖身如果要有效就逼需要做到對才行。

**Comment by PaulSu0905** [comment]
其實根據研究顯示運動前應該要用動態熱身的方式暖身，因為動態熱身在生理上可以增加肌肉温度、身體核心温度、血流量、呼吸頻率、與全身代謝反應，在運動表現上可以加快肌肉的收縮和放鬆、減短反應時間、改善肌肉力量和爆發力，並且有預防運動傷害的效果，而且靜態伸展會降低力量產生、降低運動功率輸出、反應時間及敏捷能力下降、及降低跑步速度，進而影響運動表現。但現在似乎仍有許多人甚至連體育老師都還沒接收到這樣的資訊。

**Comment by Tzuhsuannnnn** [comment]
學到了!原來暖身運動也有那麼多學問，國中時體育老師常常都是叫同學們自己上去帶，有時候自己做的暖身運動都不知道有甚麼用處，原來它甚至有可能是傷害身體的。

**Comment by Brownnnn78** [comment]
高中運動之前通常會有一些拉伸動作和慢慢跑兩圈操場，之前一直覺得好像沒有這些暖身也可以，然而大學自己去運動的時候沒有暖身真的讓身體很難伸展開來，而且在跑步的時候會更容易感受到肌肉的緊繃，而運動後的拉伸我認為更重要，不然運動完沒有拉伸都要肌肉酸痛好久，很得不償失。

**Comment by poweichen00** [comment]
近幾年來都有在更新熱身運動的方式，目前看到的資訊是運動前動態熱身，運動後靜態熱身，我覺得如果只做靜態熱身，身體並沒有暖身到太多。

**Comment by Chiuhsianglin** [comment]
我認為根據不同的運動做不同的熱身也很重要，針對需要訓練的部位加強活化，這樣對身體才不會有太大的傷害。

**Comment by Mitochondrionjessica** [comment]
老派一點的體育老師還是會堅持要做這些傷身的暖身動作，年輕一輩的體育老師基本上都會避免!

**Comment by jeff110303549** [comment]
這篇文章提到了一些常見但應該被淘汰的暖身動作，並解釋了它們可能對身體造成的負面影響。這些指出了這些動作的問題所在，並提醒我們在暖身時要注意選擇合適的動作，以避免運動傷害。

你的經歷也很值得注意，認真對待暖身是很重要的，但同時也要確保選擇的動作對身體是有益的。透過這篇文章和影片，你不僅了解了這些被淘汰的動作，還能夠更好地保護自己的身體，在運動前進行有效的暖身。希望大家都能夠在運動中保持健康和安全！
